File: 백준/Bronze/4998. 저금/저금.py
import sys
input = sys.stdin.readline

while True:
    n = list(map(float,input().split()))
    if not n:
        break

    bank = n[0]
    year = 0
    while bank <= n[2]:
        interest = bank * n[1] / 100
        bank += interest
        year += 1

    print(year)

File: 백준/Bronze/4998. 저금/저금.js
const input = require("fs")
  .readFileSync(process.platform === "linux" ? "/dev/stdin" : "./example.txt")
  .toString()
  .trim()
  .split("\n")
  .map((v) => v.split(" ").map(Number));

for (const v of input) {
  const [n, b, m] = v;
  const it = 1 + b / 100;

  let money = n;
  let res = 0;
  while (money <= m) {
    money *= it;
    res += 1;
  }
  console.log(res);
}
